plugins/english/relibrary.ts

```diff
@@ -0,0 +1,168 @@
+import { Filters } from '@libs/filterInputs';
+import { load as loadCheerio, CheerioAPI } from 'cheerio';
+import { defaultCover } from '@libs/defaultCover';
+import { NovelStatus } from '@libs/novelStatus';
+import { NovelItem } from '../../test_web/static/js';
+// import { isUrlAbsolute } from "@libs/isAbsoluteUrl";
+// import { parseMadaraDate } from "@libs/parseMadaraDate";
+
+class ReLibraryPlugin implements Plugin.PluginBase {
+  id = 'ReLib';
+  name = 'Re:Library';
+  icon = 'src/en/relibrary/icon.png';
+  site = 'https://re-library.com';
+  version = '1.0.0';
+  filters: Filters | undefined = undefined;
+
+  async popularNovels(
+    pageNo: number,
+    {
+      showLatestNovels,
+      filters,
+    }: Plugin.PopularNovelsOptions<typeof this.filters>,
+  ): Promise<Plugin.NovelItem[]> {
+    const novels: Plugin.NovelItem[] = [];
+
+    const result = await fetch(this.site + '/translations/most-popular/');
+    const body = await result.text();
+
+    const loadedCheerio = loadCheerio(body);
+    loadedCheerio('.entry-content > ol > li').each((_i, el) => {
+      let novel: NovelItem = { name: '', path: '' };
+      novel.name = loadedCheerio(el).find('h3 > a').text();
+      novel.path =
+        loadedCheerio(el).find('table > tbody > tr > td > a').attr('href') ||
+        '';
+      novel.cover = loadedCheerio(el)
+        .find('table > tbody > tr > td > a > img')
+        .attr('src');
+      if (novel.name.startsWith(this.site)) {
+        novel.name = novel.name.slice(this.site.length);
+      }
+      novels.push(novel);
+    });
+
+    return novels;
+  }
+
+  async parseNovel(novelPath: string): Promise<Plugin.SourceNovel> {
+    const novel: Plugin.SourceNovel = {
+      path: novelPath,
+      name: 'Untitled',
+    };
+    // title:		.entry-content > header.entry-header > .entry-title
+    // img:			.entry-content > table > tbody > tr > td > img
+    // tags:		.entry-content > table > tbody > tr > td > p > span > a[]
+    // synopis:		.entry-content > div.su-box > div.su-box-content
+    // chapters:	.entry-content > div.su-accordion <then> li.page_item[]
+    //
+    // TODO: get here data from the site and
+    // un-comment and fill-in the relevant fields
+
+    const result = await fetch(this.site + '/' + novelPath);
+    const body = await result.text();
+
+    const loadedCheerio = loadCheerio(body);
+
+    novel.name =
+      loadedCheerio('header.entry-header > .entry-title').text().trim() || '';
+    // novel.artist = "";
+    // novel.author = "";
+    novel.cover =
+      loadedCheerio('.entry-content > table > tbody > tr > td > img').attr(
+        'src',
+      ) || defaultCover;
+    novel.genres = (() => {
+      let genres: string[] = [];
+      loadedCheerio(
+        '.entry-content > table > tbody > tr > td > p > span > a',
+      ).each((_i, el) => {
+        genres.push(loadedCheerio(el).text().trim().replace(' ', '-'));
+      });
+      return genres.join(' ');
```

Comment:
```suggestion
      return genres.join(',');
```

Reply:
In this case, would it make sense to remove the `.replace(' ', '-')` in the line 82 ? Since I made this change only to allow the user to see which tags are multi-words and which tags are  separate tags  (for example instead of having `isekai gender bend` you would see `isekai gender-bend`).
But if you put an `,` between every tags/genres it doesn't matter.

Reply:
you just need to split genres by `,` -> ` isekai, gender bend`